cdc/model/sink.go

```diff
@@ -478,38 +478,85 @@ type RedoDDLEvent struct {
 
 // FromJob fills the values of DDLEvent from DDL job
 func (d *DDLEvent) FromJob(job *model.Job, preTableInfo *TableInfo) {
+	if d.Type == model.ActionRenameTables {
+		return
+	}
+
+	// The query for "DROP TABLE" and "DROP VIEW" statements need
```

Comment:
What about `RENAME TABLE`s?

Reply:
> What about `RENAME TABLE`s?

It is handled in `FromRenameTablesJob()`